### PR TITLE
mod checker

### PR DESCRIPTION
**Pull request by qatroxx**
mod checker
/wrench

### CODE REVIEW

**Comment by demonicbae** [comment]
Cool bro... But i also had mod detector but this one is cool too!

**Comment by demonicbae** [comment]
interface/atomic_button.rttex For atomic too :)

File: proxy/events.cpp [hunk 1]
```diff
@@ -314,6 +323,18 @@ bool events::in::variantlist(gameupdatepacket_t* packet) {
                 }
                 if (var.get("mstate") == "1" || var.get("smstate") == "1")
                     ply.mod = true;
+                variantlist_t gladiator{ "OnAddNotification" };
+                    gladiator[1] = "interface/science_button.rttex";
+                    gladiator[2] = "Mod here!";
+                    gladiator[3] = "audio/hub_open.wav", 0;
+                    g_server->send(true, gladiator, -1);
+                }
```
Comment (ama6nen) [inline proxy/events.cpp:331]: Will probably break compiling, seems like trailing }

File: proxy/events.cpp [hunk 2]
```diff
@@ -314,6 +323,18 @@ bool events::in::variantlist(gameupdatepacket_t* packet) {
...
+                    variantlist_t gladiator{ "OnAddNotification" };
+                    gladiator[1] = "interface/science_button.rttex";
+                    gladiator[2] = "Mod here!";
+                    gladiator[3] = "audio/hub_open.wav", 0;
+                    g_server->send(true, gladiator, -1);
+                }
```
Comment (ama6nen) [inline proxy/events.cpp:337]: Will probably break compiling, seems like trailing }
And also why is the same code here twice

File: proxy/events.cpp [hunk 3]
```diff
@@ -126,6 +126,15 @@ bool events::out::generictext(std::string packet) {
...
             return true;
+        } else if (find_command(chat, "wrench ")) {
+            std::string name = chat.substr(6);
+            std::string username = ".";
+            for (auto& player : g_server->m_world.players) {
+                auto name_2 = player.name.substr(2);
+                std::transform(name_2.begin(), name_2.end(), name_2.begin(), ::tolower);
+                    g_server->send(false, "action|wrench\n|netid|" + std::to_string(player.netid));
+            }
+            return true;
```
Comment (ama6nen) [inline proxy/events.cpp:137]: Was already added from another merge

**Pull request closed without merging**